conans/test/functional/command/copy_packages_test.py

```diff
@@ -40,3 +42,43 @@ class Pkg(ConanFile):
         client.run("copy Hello0/0.1@lasote/stable pepe/alpha")
         pkgdir = client.cache.packages(ConanFileReference.loads("Hello0/0.1@pepe/alpha"))
         self.assertFalse(os.path.exists(pkgdir))
+
+    @parameterized.expand([(True, ), (False,)])
+    def test_copy_exports_sources_with_revision_command(self, revision_enabled):
+        server = TestServer()
+        client = TestClient(servers={"default": server},
+                            users={"default": [("lasote", "mypass")]},
+                            revisions_enabled=True)
+
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    exports_sources = "*"
+"""
+
+        client.save({"conanfile.py": conanfile,
+                     "myfile.txt": "my data!"})
+        client.run("export . pkg/0.1@lasote/stable")
+        client.run("upload pkg/0.1@lasote/stable")
+
+        client2 = TestClient(servers={"default": server},
+                             revisions_enabled=revision_enabled)
+        client2.run("info pkg/0.1@lasote/stable")
```

Comment:
Useless or check that it doesn't fail? or...

Reply:
It is the only way to install dependencies without installing binaries too. I didn't create binaries, it is not needed for reproducing this, to keep the test as minimal and as fast as possible.

I have added a couple of comments to clarify it. 

A different story is if we wanted to enable some way to ``conan install --no-binaries`` or something like that, not under the ``conan info`` command.